data/json/items/comestibles.json

```diff
@@ -390,6 +390,28 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "soymilk",
+        "name" : "soymilk",
+        "name_plural": "soymilk",
+        "weight" : 260,
+        "color" : "white",
+        "spoils_in" : 8,
+        "container" : "jug_plastic",
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 30,
+        "healthy" : 1,
+        "nutrition" : 4,
```

Comment:
If you're copying most of the data from cow milk, you should use `"copy-from": "milk"` and only include changes from cowmilk here. Changes like material, description, name etc. - everything that is the same should be left unspecified and handled by "copy-from".